tests/core_tests.py

```diff
@@ -1231,6 +1231,22 @@ def test_get_column_names_from_metric(self):
             "my_col"
         ]
 
+    @mock.patch.dict(
```

Comment:
I added a test for `stop_dashboard_queries`. @bkyryliuk Do you think it make sense?